src/elgamal.rs

```diff
@@ -215,6 +216,10 @@ impl ExponentElGamal {
         _sk: &ElGamalPrivateKey,
     ) -> Result<BigInt, ElGamalError> {
         //TODO
```

Comment:
This is not a traditional EG decrypt. Can you move it to the test module?

Reply:
yep